packet/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -63,13 +63,7 @@ variable "facility" {
 variable "os_channel" {
   type        = "string"
   default     = "stable"
-  description = "Flatcar Linux channel to install from (stable, beta, alpha)"
-}
-
-variable "os_version" {
```

Comment:
I guess it is not an option (and if it is, it won't be in a first stage) to have several versions and be able to choose from, right?

Reply:
Looking at other OS-es on Packet, it is highly unlikely they will offer multiple versions for each channel. We can ask, but I don't think they will be able to offer a lot of history considering Flatcar's rapid release cadence.

Reply:
Totally agreed. And I think it is not an issue AFAIK. We can solve it later, if we have a good use case :)